scripts/fixGenerated.cjs

```diff
@@ -0,0 +1,226 @@
+/* eslint-disable quotes */
+
```

Comment:
Can you add some comments, explaining the reasons for this file and what it's doing